utils/mediaupload.js

```diff
@@ -56,14 +57,16 @@ export function mediaUpload( filesList, onFileChange, allowedType ) {
 }
 
 /**
- * @param {File} file Media File to Save.
+ * @param {File}   file   Media File to Save.
+ * @param {number} postId Optional post id to associate with.
  *
  * @return {Promise} Media Object Promise.
  */
-function createMediaFromFile( file ) {
+function createMediaFromFile( file, postId ) {
 	// Create upload payload
 	const data = new window.FormData();
 	data.append( 'file', file, file.name || file.type.replace( '/', '.' ) );
+	data.append( 'post', postId );
```

Comment:
Could you please add a test case for this to `utils/test/mediaupload.js`?

Reply:
>  It would be nice if we mocked `wp.apiRequest` and asserted that the POST request we build is what we expected. Up to you, though.

I'm going to hold off for now. I think e2e tests will be more durable in the long term, but I'm still ramping up on where things currently stand.